src/d3a/setup/strategy_tests/home_hybrid_pv_ess_load.py

```diff
@@ -1,72 +0,0 @@
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
-"""
-from d3a.models.appliance.switchable import SwitchableAppliance
-from d3a.models.area import Area
-from d3a.models.strategy.storage import StorageStrategy
-from d3a.models.strategy.commercial_producer import CommercialStrategy
-from d3a.models.strategy.load_hours import LoadHoursStrategy
-from d3a.models.appliance.simple import SimpleAppliance
-from d3a.models.appliance.pv import PVAppliance
-from d3a.models.strategy.pv import PVStrategy
-
-'''
-This setup file is being modified to mimic a typical
-modern household having a hybrid PV-ESS system.
-Bug: The PV should be selling power cheaper than the Commercial Producer, but it's not selling
-'''
-
-
-def get_setup(config):
-    config.read_market_maker_rate(30)
-    area = Area(
-        'Grid',
-        [
-            Area(
-                'House 1',
-                [
-                    Area('H1 General Load', strategy=LoadHoursStrategy(avg_power_W=100,
-                                                                       hrs_per_day=24,
-                                                                       hrs_of_day=list(
-                                                                           range(0, 24)),
-                                                                       final_buying_rate=14),
-                         appliance=SwitchableAppliance()),
-                    Area('H1 Storage1', strategy=StorageStrategy(risk=10,
-                                                                 initial_capacity_kWh=0.6,
-                                                                 initial_rate_option=2,
-                                                                 energy_rate_decrease_option=2,
-                                                                 energy_rate_decrease_per_update=7,
-                                                                 battery_capacity_kWh=1.2,
-                                                                 max_abs_battery_power_kW=5,
-                                                                 break_even=(12, 17.01)),
-                         appliance=SwitchableAppliance()),
-                    Area('H1 PV', strategy=PVStrategy(panel_count=4,
-                                                      final_selling_rate=5,
-                                                      initial_rate_option=2,
-                                                      energy_rate_decrease_option=2,
-                                                      energy_rate_decrease_per_update=7),
-                         appliance=PVAppliance()),
-
-                ]
-            ),
-            Area('Commercial Energy Producer',
-                 strategy=CommercialStrategy(energy_rate=15),
-                 appliance=SimpleAppliance()),
-        ],
-        config=config
-    )
-    return area
```

Comment:
I would suggest to keep this setup file, as it depicts an smart home self balancing itself and then purchasing the residual electricity from utility.

Reply:
Ok, but how should we tackle this? The setup file is not up to date (some parameter have to be removed) but i dont know the expected outcome. Do YOU know?

Reply:
Im not sure either, should there be a rule of deleting the setup file which are not part of any integration test?
May be @spyrostz has better opinion on it

Reply:
I would suggest to keep this and document this better (maybe via an integration test). The reason for that lies on the comments of the setup file. The reason why the one that created this setup file has it here, is because the commercial producer offers arrive to the load sooner than the PVs. This needs to be described in Gherkin language in an integration test, so that at least we can understand why this setup file is there. Since this is not in the context of this story, you can reset the file and open a new enhancement story for adding an integration test for this file.

Reply:
Ok! Will include an integrationtest.

Reply:
@faizan2590 @spyrostz 
I am currently trying to reenact this setup file with the new parameter set. I dont understand the break_even set to (12, 17.01) This way the storage never sells to the load.
I still need a motivation for this setup file. Spyros wrote: "because the commercial producer offers arrive to the load sooner than the PVs" I dont know how to reenact this. Can you help me?
Faizan was writing that it should be an example of a "smart home self balancing itself" I can do that. 
I just realised that Spyros wrote that this should be done in a different story. IMO we should deal with it now or delete the setup file. Otherwise this story will never be dealt with.

Reply:
@faizan2590 @spyrostz Sorry, will skip this one and keep the old version of the setup file in here (commented out with a TODO message) IMO we have more pressing stuff to do. Thanks anyway!

Reply:
@hannesdiedrich regarding this:
> Otherwise this story will never be dealt with.
every story that ends up in the backlog should be dealt with eventually. Even if it takes a long time, this can act as documentation so that we know what this setup file is about. 

The issue described in the comments is quite clear. According to that, Load buys energy only from the Commercial Producer and not from the PV, even though the PV is cheaper. Did you notice where does the load buy its energy from ? If it is the Commercial Producer, then this "bug/behavior" is still valid. The reason why this behavior appears is again easy to understand, because the Commercial Producer will place an offer before the PV places one. Due to the fact that offers propagate immediately, Commercial Producer's offer will reach the Load before PV even manages to place an offer. Therefore, the behavior can be explained. Hope this is clear.

Reply:
Thanks @spyrostz!
Yes, this makes sense, BUT. I can not reproduce with the current master version.
For making the setup file run, I have to delete all deprecated parameters and exchange break_even with final_selling_rate and final_buying_rate. When I run this setup file, the load buys energy from the PV perfectly and the storage charges only with the pv energy. No behaviour seen, like mentioned in the comment on the top. 
Makes sense, right? 
Shall I introduce an integrationtest that checks for this behaviour?
I still not think that this was the purpose of the original setup file (judging from the comment in it).

Reply:
The more we discuss about it the more I tend to think that we should open a separate small story for that, that clarifies the expected behavior and writes an integration test for this. So, feel free to ignore this setup file in your PR, and I will open a story that deals with this.